From 496633d1667ff9761b7ecbc3004f0fb2287d8ee9 Mon Sep 17 00:00:00 2001
From: Christian Bruel <christian.bruel@foss.st.com>
Date: Tue, 12 Dec 2023 13:16:04 +0100
Subject: [PATCH 0782/1141] arm64: defconfig: add CONFIG_IGC in
 fragment-02-defconfig-addons

Enable IGC Ethernet to use PTM support.

Signed-off-by: Christian Bruel <christian.bruel@foss.st.com>
Reported-by: Alexandre Torgue <alexandre.torgue@foss.st.com>
Change-Id: I31c3f52aada2c5d75b8e3406f1bc54782e403af3
Reviewed-on: https://gerrit.st.com/c/mpu/oe/st/linux-stm32/+/345916
Reviewed-by: Fabrice GASNIER <fabrice.gasnier@foss.st.com>
ACI: CIBUILD <MDG-smet-aci-builds@list.st.com>
ACI: CITOOLS <MDG-smet-aci-reviews@list.st.com>
Domain-Review: Fabrice GASNIER <fabrice.gasnier@foss.st.com>
---
 arch/arm64/configs/fragment-02-defconfig-addons.config | 1 +
 1 file changed, 1 insertion(+)

--- a/arch/arm64/configs/fragment-02-defconfig-addons.config
+++ b/arch/arm64/configs/fragment-02-defconfig-addons.config
@@ -34,6 +34,7 @@ CONFIG_HDMI=y
 CONFIG_I2C_RPMSG=m
 CONFIG_I2C_STM32F7=y
 CONFIG_I3C=y
+CONFIG_IGC=m
 CONFIG_IIO_STM32_LPTIMER_TRIGGER=m
 CONFIG_IIO_STM32_TIMER_TRIGGER=m
 CONFIG_INPUT_TOUCHSCREEN=y
